packages/backend/src/asset/service.ts

```diff
@@ -21,10 +23,15 @@ export interface UpdateOptions {
   withdrawalThreshold: bigint | null
   liquidityThreshold: bigint | null
 }
+export interface DeleteOptions {
+  id: string
+  deletedAt: string
```

Comment:
```suggestion
  deletedAt: Date
```
ObjectionJS can handle the casting, I believe we don't need to call `.toISOString` manually before inserting into the DB